src/main/java/teammates/ui/webapi/action/SearchInstructorsAction.java

```diff
@@ -49,7 +49,8 @@ public ActionResult execute() {
             InstructorData instructorData = new InstructorData(instructor);
             instructorData.addAdditionalInformationForAdminSearch(
                     StringHelper.encrypt(instructor.getKey()),
-                    getInstituteFromGoogleId(instructor.getGoogleId()));
+                    getInstituteFromGoogleId(instructor.getGoogleId()),
+                    instructor.getGoogleId());
             instructorData.hideInformationForSearch();
```

Comment:
Only admins can search instructors, and we have no reason to hide anything from admin. This line can be removed.